c7n/resources/appelb.py

```diff
@@ -201,7 +201,7 @@ def process(self, resources, event=None):
         name_id_map = {}
         resource_map = {}
 
-        wafs = self.manager.get_resource_manager('waf-regional').resources()
+        wafs = self.manager.get_resource_manager('waf-regional').resources(None, False)
```

Comment:
augment=False instead of passing None, False

Reply:
Ah thanks, that reads much better. Resolved in f9d6ed0.